eftcamb/11_EFTCAMB_stability.f90

```diff
@@ -252,8 +180,8 @@ function EFTStabilityComputation(a)
 !        ! All the coefficients should not be Nan. This can happen for strange values of the parameters
 !        ! for which a division by zero may occur.
 !
-!        EFT_HaveNan = IsNaN(EFTAT).or.IsNaN(EFTpiA1).or.IsNaN(EFTpiA2).or.IsNaN(EFTpiB1).or.IsNaN(EFTpiB2)&
-!            & .or.IsNaN(EFTpiC).or.IsNaN(EFTpiD1).or.IsNaN(EFTpiD2).or.IsNaN(EFT_kinetic).or.IsNaN(EFT_gradient)
+!        EFT_HaveNan = IsNaN(eft_cache%EFTAT).or.IsNaN(eft_cache%EFTpiA1).or.IsNaN(eft_cache%EFTpiA2).or.IsNaN(eft_cache%EFTpiB1).or.IsNaN(eft_cache%EFTpiB2)&
```

Comment:
We can add the NaN check to the whole cache as a method of the cache. In this way we can check all the quantities to make sure that we do not have NaN's somewhere...